test/pylib/rest_client.py

```diff
@@ -374,13 +374,14 @@ async def backup(self, node_ip: str, ks: str, table: str, tag: str, dest: str, b
                 params[key] = value
         return await self.client.post_json(f"/storage_service/backup", host=node_ip, params=params)
 
-    async def restore(self, node_ip: str, ks: str, cf: str, dest: str, bucket: str, prefix: str, sstables: list[str], scope: str = None) -> str:
+    async def restore(self, node_ip: str, ks: str, cf: str, dest: str, bucket: str, prefix: str, sstables: list[str], scope: str = None, primary_replica_only=False) -> str:
         """Restore keyspace:table from backup"""
         params = {"keyspace": ks,
                   "table": cf,
                   "endpoint": dest,
                   "bucket": bucket,
-                  "prefix": prefix}
+                  "prefix": prefix,
+                  "primary_replica_only": "true" if primary_replica_only else "false"}
```

Comment:
nit: "primary_replica_only": str(primary_replica_only).lower()

Reply:
won't work, `None` is an option too.